Export/UI: improve "NLA Strips" option name/description

scurest opened the pull request:
(Belated) followup to https://github.com/KhronosGroup/glTF-Blender-IO/pull/764#issuecomment-551403017.

The name and tooltip for the "NLA Strips" option is currently basically impossible to understand unless you already know what it does. We should try to improve it.

How does this look? I tried to make it both descriptive and concise.

Alternative name idea: `Multiple Animations (organized by NLA)`

scurest commented:
Alternative text idea: "When on, create a glTF animation named 'Foo' by pushing all actions that should play onto NLA tracks named 'Foo'."

scurest commented:
cc @donmccurdy

julienduroure commented:
I like this new description, that reflects really better the feature. If Don validates it too, we need to update the documentation too.

donmccurdy commented:
Perhaps "Group by" rather than "Organize by"? Otherwise this looks good!

scurest commented:
I'm good with "Group by". Should I go with that?

scurest commented:
I changed it to "Group by" and updated the option name in the docs.

The docs about animation (all the stuff about "NLA Strip animations") could probably be reworked too. Basically I wrote those in reverse: given that the "NLA Strips" option meant "Export NLA Strip animations", I tried to write a definition for "NLA Strip animation" that would explain what that meant. But I'm not gonna try to change it in this PR.

emackey commented:
One suggestion, otherwise looks good to me too.

scurest commented on addons/io_scene_gltf2/__init__.py:
```diff
@@ -274,8 +274,12 @@ def __init__(self):
-        description='Export NLA Strip animations',
+        name='Group by NLA Track',
+        description=(
+            "When on, two actions become part of the same glTF animation if they're\n"
```
Done.
